video/out/wayland_common.c

```diff
@@ -1046,6 +1043,7 @@ int vo_wayland_init (struct vo *vo)
     }
 
     vo->event_fd = wl->display.display_fd;
```

Comment:
Also, this should be removed. Your new code won't actually be called: https://github.com/mpv-player/mpv/blob/master/video/out/vo.c#L633